flow/connectors/postgres/cdc.go

```diff
@@ -240,53 +241,42 @@ func (p *PostgresCDCSource) consumeStream(
 					// tableName here is destination tableName.
 					// should be ideally sourceTableName as we are in PullRecords.
 					// will change in future
-					isFullReplica := req.TableNameSchemaMapping[tableName].IsReplicaIdentityFull
```

Comment:
looks like `isFullReplica` is unintentionally removed?

Reply:
reverted